README.md

```diff
@@ -20,7 +20,7 @@ MongoDB is a great NoSQL and schema-less database, but if already have data in d
 ## How to use it
 
 1. create a fold named `migrations`
-2. create a python file with name in form of `TIMESTAMP_description.py` , i.e.`20160320145400_description.py`, otherwise migration file won't be found.
+2. create a python file with name in form of `TIMESTAMP_description.py` , where `TIMESTAMP` is in the form `YYYYMMDDhhmmss` and `description` is free text (i.e.`20160320145400_description.py`), otherwise migration file won't be found.
```

Comment:
No, it does not need to be in this format. I used 1_desc, and 2_desc and this is also a correct timestamp format and I think more useful.